runtimes/js/src/error.rs

```diff
@@ -0,0 +1,72 @@
+use crate::log::parse_js_stack;
+use encore_runtime_core::api;
+use napi::{Env, JsUnknown, Status};
+
+pub fn coerce_to_api_error(env: Env, val: napi::JsUnknown) -> Result<api::Error, api::Error> {
```

Comment:
Should the return type just be `api::Error`? Is the Ok/Err distinction useful?

Reply:
We can change it to that, its just nice to be able to know if the coerce succeeded or not imo, even though both success and fail is the same type.

As we use it today we dont care if it succeeded or not, so I dont have strong opinions about it

Reply:
Fair, can keep it as-is